hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -715,14 +717,52 @@ ppResult name (Report tests discards result) =
         ppDiscardCount discards <>
         ", passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
 
     OK ->
       pure . icon SuccessIcon '✓' . WL.annotate SuccessHeader $
         ppName name <+>
         "passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
+
+ppClassification :: Classification -> Doc Markup
+ppClassification (Classification cls total)
+  | Map.null cls = mempty
+  | otherwise = (<>) WL.linebreak $ WL.indent 4 . WL.align . WL.vsep $
+    (\(k, v) -> WL.text $ show (occurrenceRate v total) <> "% " <> k) <$> Map.toList cls
+
+occurrenceRate :: Classifier -> Integer -> Double
+occurrenceRate (Classifier _ occurrences) total =
+  let
+    percentage' :: Double
+    percentage' = fromIntegral occurrences / fromIntegral (total - 2) * 100
```

Comment:
No primes please, I usually use decimal numbers instead as they are harder to miss when reading the code. 

`x0` `x1` `x2` vs `x` `x'` `x''`

Although here the prime doesn't seem necessary? I guess you are using it to signify that this is the "next percentage".